app/models/feedback.rb

```diff
@@ -7,18 +7,23 @@ class Feedback
     bug_report: %i[case_number event outcome email]
   }.freeze
 
+  SENDING_SERVICES = [
+    SurveyMonkeySender,
+    ZendeskSender
+  ].freeze
+
   attr_accessor :email, :referrer, :user_agent, :type,
                 :event, :outcome, :case_number,
                 :task, :rating, :comment, :reason, :other_reason, :response_message
 
   validates :type, inclusion: { in: FEEDBACK_TYPES.keys.map(&:to_s) }
   validates :event, :outcome, presence: true, if: :bug_report?
 
-  def initialize(attributes = {}, sender = nil)
+  def initialize(sender = nil, attributes = {})
```

Comment:
New commit now submits :sender as a part of the existing attributes hash rather than on its own. Validation only required slight tweaking, as a non-existent :sender attribute is nil, which can still be checked against the list of acceptable senders